src/shared/components/ncContextResource/ResourceSharees.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+	<div class="row space-B">
+		<div class="col-4">
+			{{ t('tables', 'Share with accounts') }}
+		</div>
+		<NcSelect v-model="preExistingSharees" style="width: 100%;" :loading="loading" :options="options"
+			:placeholder="getPlaceholder()"
+			:searchable="true" :get-option-key="(option) => option.key"
+			label="displayName"
+			:aria-label-combobox="getPlaceholder()" :user-select="true"
+			:close-on-select="false"
+			:multiple="true"
+			@search="asyncFind" @input="addShare">
```

Comment:
I notice that we already have several `NcSelect`s that search for users/groups duplicated across other pages. Thinking about how to consolidate into just one and reuse

Reply:
That would be very nice indeed

Reply:
follow up though?

Reply:
@enjeck Could you file a follow up issue